src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/Referral.kt

```diff
@@ -21,6 +21,12 @@ import javax.validation.constraints.NotNull
 @Entity
 @Table(indexes = arrayOf(Index(columnList = "created_by_id")))
 data class Referral(
+  // assigned referral fields
+  var assignedAt: OffsetDateTime? = null,
+  @ManyToOne(fetch = FetchType.LAZY) var assignedBy: AuthUser? = null,
```

Comment:
Should we make these eager and they are used most of the time by the DTOs?
@Fetch(value = FetchMode.JOIN)

Reply:
done